common/redux-logger.js

```diff
@@ -0,0 +1,4 @@
+module.exports = store => next => action => {
+  console.log("ACTION", action); // eslint-disable-line no-console
```

Comment:
this is specifically a logger (i.e. logging redux state to console), so we do need it